python/simulation/fake_raft.py

```diff
@@ -0,0 +1,509 @@
+"""
+Makes a raft's worth of image files based on a user-specified single-sensor
+image file, the raft designator, and the sensor type (ITL or E2V, which 
+probably could be guessable from the single-sensor image).  The test type
+probably also needs to be specified (for the file naming)
+The data in the nine output images are the same as in the input image.  What
+is new is the raft coordinate-related header keywords.  These are derived
+from the specification in LCA-13501.
+"""
+
+from __future__ import print_function, absolute_import, division
+#import os
+#import copy
+import numpy as np
+import astropy.io.fits as fits
+import os
+import yaml
+
+import fnmatch
+import lcatr.schema
+from DataCatalog import DataCatalog
+
+import siteUtils
+
+
+def sensor_type_from_id(sensor_id):
+    """  Get the sensor type ( e2v-CCD or ITL-CCD ) from the sensor_id string
+    """
+    stype =  sensor_id[0:7]
+    if stype == 'E2V-CCD':
+        stype = 'e2v-CCD'
+    return stype
+
+
+def make_datacat_path(root_folder,sensor_id,process_name,test_version="v0"):
+    """ Build the data catalog path for a particular test on a particular sensor
+
+    This appear to be <root_folder>/<sensor_type>/<sensor_id>/<process_name>/<test_version>
```

Comment:
I've changed this to work with the new directory format and changed the query accordingly